docs/notebook_validation.py

```diff
@@ -35,6 +35,10 @@ def validate(notebook_filename, available_backends):
         match = re.search('--target ([^ ]+)', notebook_content)
         if match and (match.group(1) not in available_backends):
             return False
+    for notebook_content in lines:
+        match = re.search('(config.update_option)', notebook_content)
```

Comment:
Just curious ... what is `config.update_opton`? Maybe I'll keep reading the rest of the files and find out, but I figured I'd ask anyway.

Reply:
So does that mean we are disabling all validation of this new tutorial, even though that line is disabling GPU support? If so, why does validation need to be disabled if no GPUs are required on the validation runner?